src/index.tsx

```diff
@@ -130,9 +131,7 @@ const {
   function createClosure ({ listeners, app }: Instance): Closure {
     return {
       sendData (name: string, payload: any) {
-        if (!app.ports[name]) {
-          console.warn(errors.missingPort(name))
-        } else {
```

Comment:
I think for the time being I would prefer to keep this warning here. Although in the case where you are choosing to only use the prop for flags, this hurts the developer experience for the normal use case of using ports. I think this warning is important for people that might have attempted to setup ports and done so improperly

Reply:
I removed the commit introducing this.